tests/integration/utils2.go

```diff
@@ -196,17 +196,24 @@ func init() {
 	}
 }
 
-// AssertPanicAndSkipChangeDetection asserts that the code of function actually panics,
+// AssertPanic asserts that the code inside the specified PanicTestFunc panics.
```

Comment:
nitpick(non-blocking): Rename suggestion
```suggestion
// AssertPanicOrSkip asserts that the code inside the specified PanicTestFunc panics.
```

Reply:
:) I do disagree here, although I think my views over time have shifted, and I might have been the one to originally ask for the longer name.  Will leave as `AssertPanic` unless anyone else requests the change before I rebase and merge.